docs/CSSLoader/Features.md

```diff
@@ -328,7 +330,44 @@ The above will inject shared.css and shared2.css into the Steam Big Picture Mode
 A special tab mapping name called "default" can be used to create a custom default when no tabs are specified. This will result in the same behaviour as the first example.
 
 
-### 🧭 File watcher
+### :material-hammer-wrench: Nav Patching
+*Requires Manifest version 9+*
+
+When hiding elements that are selectable by controller navigation, it most of the time is still selectable by controller navigation, despite it being hidden. The 'Nav Patch' fixes this by checking for `display: none;` on .Focusable elements, and skipping those in the controller navigation routine.
```

Comment:
I'd change this to "When using `display: none` on elements, sometimes they are still selectable by the controller navigation. The 'Nav Patch'  blah blah